src/components/RegistryInfoContent/index.ts

```diff
@@ -1 +1 @@
-export { default } from './RegistryInfoContent';
```

Comment:
please try to keep the standard 😉

Reply:
same as https://github.com/verdaccio/ui/pull/229#discussion_r339846597 wondering why we want to use default exports?